tests/utils_tests/test_timezone.py

```diff
@@ -1,14 +1,38 @@
 import datetime
+import unittest
 from unittest import mock
 
 import pytz
 
+try:
+    import zoneinfo
+except ImportError:
+    try:
+        from backports import zoneinfo
+    except ImportError:
+        zoneinfo = None
+
 from django.test import SimpleTestCase, override_settings
 from django.utils import timezone
 
-CET = pytz.timezone("Europe/Paris")
+PARIS = pytz.timezone('Europe/Paris')
```

Comment:
Diff will be smaller without this unnecessary change.

Reply:
I suppose it's somewhat unnecessary, but it is confusing because `CET` represents standard time, but `pytz.timezone("Europe/Paris")` is both `CEST` and `CET`.  The nomenclature is further confused because `CEST` is used elsewhere to represent the _fixed_ offset of Central European Summer Time.

True, changing the name of this variable has no _instrumental_ value, but I think it improves the readability of the tests, especially considering the other changes accompanying it.

Might be worth a later PR to go through and make the `s/CET/PARIS/g` change in this file again.